content/docs/notebooks/why-use-jupyter-notebook.md

```diff
@@ -1,17 +1,16 @@
 +++
-title = "Using Jupyter notebooks in kubeflow"
-description = "Why use Jupyter notebooks in Kubeflow"
+title = "Using Jupyter Notebooks in Kubeflow"
+description = "Why use Jupyter Notebooks in Kubeflow"
 weight = 30
 +++
 
-Why would you use Jupyter networks in Kubeflow?
 
-The benefit of integrating Jupyter notebooks service in Kubefflow is that:
+The benefit of integrating Jupyter Notebooks service in Kubeflow is that:
```

Comment:
I find the addition of `service` confusing. The Jupyter Notebook Kubeflow component is a Kubernetes Service?

Reply:
+1 that the usage of "service" is confusing here.